.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,43 @@
+name: Bug report
+description: Create a report to help us improve.
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      placeholder: "<!-- A clear and concise description of what the bug is. -->"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      placeholder: |
+        1. 
+        2. 
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      placeholder: "<!-- A clear and concise description of what you expected to happen. -->"
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Screenshots
+      placeholder: "<!-- If applicable, add screenshots to help explain your problem. -->"
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
```

Comment:
Should be `System information` instead (and it should resolve the issue on the screenshot above)

Reply:
Apologies. That wasn't deliberate.